src/renderer/fonts/caching_shaper.rs

```diff
@@ -97,6 +99,26 @@ impl CachingShaper {
         }
     }
 
+    pub fn update_linespace(&mut self, linespace: &u64) {
+        debug!("Updating linespace: {}", linespace);
+
+        let font_key = FontKey {
+            italic: false,
+            bold: false,
+            family_name: self.options.primary_font(),
```

Comment:
Hm... this'd mean that `linespace` only applies to the first font, not to any fallback fonts. But given the following scenario:

1. `set guifont=Hack,Roboto_Mono`
2. `set linespace=4`
3. `set linespace=Roboto_Mono,Hack`

then `Hack` would have the lineheight of `4`, but `Roboto_Mono` the default of `0`. Is that intended? (I did not test this, only a mindgame. Possible that I missed something. I should go test this, but I need to go in 7 minutes xD)

Reply:
Good points. And this should definitely be tested! 

No worries. It can sit here as long as needed.

Reply:
False alarm, can't seem to actually reproduce this.